ziti/cmd/create/create_config_environment.go

```diff
@@ -187,47 +188,47 @@ func NewCmdCreateConfigEnvironment() *cobra.Command {
 		"environment variables are used. All settings have default values but can be manually set to override " +
 		"the config output.\n\nThe following environment variables can be set to override config values " +
 		"(current value is displayed):\n")
-	sb.WriteString(fmt.Sprintf("%-40s %-50s %s\n", constants.ZitiHomeVarName, constants.ZitiHomeVarDescription, data.ZitiHome))
```

Comment:
Please double check tidying here.  This is the environment usage hint output and the removed strings didn't seem to be doing anything (not printed).